res/css/views/elements/_ToggleSwitch.scss

```diff
@@ -24,6 +24,8 @@ limitations under the License.
 
     background-color: $togglesw-off-color;
     opacity: 0.5;
+
+    cursor: unset;
```

Comment:
where is the cursor being set from when disabled? It feels like this is the wrong place to be disabling it if it's showing up from somewhere else.

Also, this feels unrelated?

Reply:
> where is the cursor being set from when disabled? It feels like this is the wrong place to be disabling it if it's showing up from somewhere else.

Every `ToggleSwitch` is wrapped in `AccessibleButton`, and AB sets `cursor: pointer` as a base. Perhaps a better fix though is to apply the AB disabled class as well when disabled...?

> Also, this feels unrelated?

Yeah, fair enough, just happened to notice it because the room directory publish setting is toggle. I'll remove from here.